Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/utils/MotionListener.java

```diff
@@ -0,0 +1,88 @@
+package org.dolphinemu.dolphinemu.utils;
+
+import android.content.Context;
+import android.hardware.Sensor;
+import android.hardware.SensorEvent;
+import android.hardware.SensorEventListener;
+import android.hardware.SensorManager;
+
+import org.dolphinemu.dolphinemu.NativeLibrary;
+import org.dolphinemu.dolphinemu.NativeLibrary.ButtonType;
+
+public class MotionListener implements SensorEventListener
+{
+  private SensorManager mSensorManager;
+  private Sensor mAccelSensor;
+  private Sensor mGyroSensor;
```

Comment:
Do we want to mark those as `final`? They're set in the ctor and _shouldn't_ change after the fact.